src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
I guess I'm okay with this:
```css
.icon-success:before,
.icon-warning:before,
.icon-danger:before,
.icon-info:before {
    /*Icon font has no generic fallback*/
    /*noinspection CssNoGenericFontName*/
    font-family: 'Glyphicons Halflings';
    display: inline-block;
    padding-right: .5em;
    vertical-align: top;
}
```
It'd look nicer with padding using spaces though, which still works even though it differs from what IntelliJ inserts when you select `Suppress for statement` from the IntelliJ menu:
```css
.icon-success:before,
.icon-warning:before,
.icon-danger:before,
.icon-info:before {
    /* Icon font has no generic fallback */
    /* noinspection CssNoGenericFontName */
    font-family: 'Glyphicons Halflings';
    display: inline-block;
    padding-right: .5em;
    vertical-align: top;
}
```
Mixing both styles looks weird though:
```css
.icon-success:before,
.icon-warning:before,
.icon-danger:before,
.icon-info:before {
    /* Icon font has no generic fallback */
    /*noinspection CssNoGenericFontName*/
    font-family: 'Glyphicons Halflings';
    display: inline-block;
    padding-right: .5em;
    vertical-align: top;
}
```
@LiHaoTan are you okay with the middle option? Or do you think that's unreliable as adding spaces might cause the suppression to break if IntelliJ changes their implementation in a future version?